packages/boba/bundler/src/BundlerServer.ts

```
@@ -119,8 +166,52 @@ export class BundlerServer {
           params[1]
         )
         break
+      // custom errors post bedrock!
+      // case 'eth_estimateUserOperationGas':
+      //   result = await this.methodHandler.estimateUserOperationGas(
```

shouldn't we instead return an error that informs the user that the method exists but is not currently supported?

---

otherwise it's just the generic not supported/not exists error

---

i think this is the same endpoint you are working on? so we can add this back? "estimateUserOperationGas"

---

yeah, that can be fixed later on. for now I added a new error type and throw an error for not supported